Part of this upstream commit:

commit 11bdc5eed94da167643a05a59b88dfc3c2d7c3bb
Author: Eduardo Silva <eduardo@calyptia.com>
Date:   Mon Oct 10 21:03:06 2022 -0600

    out_stackdriver: adjust test formatter callback params
    
    Signed-off-by: Eduardo Silva <eduardo@calyptia.com>

diff --git a/plugins/out_stackdriver/stackdriver_conf.c b/plugins/out_stackdriver/stackdriver_conf.c
index 26b9cb43dc8d7dad..052c01b49ca7aa38 100644
--- a/plugins/out_stackdriver/stackdriver_conf.c
+++ b/plugins/out_stackdriver/stackdriver_conf.c
@@ -176,13 +176,12 @@ static int read_credentials_file(const char *cred_file, struct flb_stackdriver *
 
 static int parse_configuration_labels(struct flb_stackdriver *ctx)
 {
-    int ret;
     char *p;
     flb_sds_t key;
     flb_sds_t val;
+    struct flb_kv *kv;
     struct mk_list *head;
     struct flb_slist_entry *entry;
-    msgpack_object_kv *kv = NULL;
 
     if (ctx->labels) {
         mk_list_foreach(head, ctx->labels) {
@@ -212,11 +211,11 @@ static int parse_configuration_labels(struct flb_stackdriver *ctx)
                 return -1;
             }
 
-            ret = flb_kv_item_create(&ctx->config_labels, key, val);
+            kv = flb_kv_item_create(&ctx->config_labels, key, val);
             flb_sds_destroy(key);
             flb_sds_destroy(val);
 
-            if (ret == -1) {
+            if (!kv) {
                 return -1;
             }
         }
